atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -90,11 +95,21 @@
 import com.palantir.lock.v2.TimelockService;
 
 // TODO (jkong): Assert stricter properties on the types of ranged deletes that are created.
```

Comment:
I think at the time I wasn't too happy that we test absence by just saying "we can't read the value", since theoretically a legacy sweeper that inserted single-cell deletes could pass.

Is this realistically going to happen? Probably not... I'll remove it on the basis that we don't plan on doing it.